### PR TITLE
Remove Disconnect customizations and sync with latest Mozilla upstream.

### PR DESCRIPTION
**Pull request by fmarier**
This is the first part of #34.

I prepared a new list by reverting [our customizations](https://github.com/brave/tracking-protection/commit/f1de4fb8ffbdedfdc2602b1e4f857e1547ae169b) and that gave me the following diff:

```
--- TrackingProtection-after.txt	2019-04-09 21:42:14.394777225 -0700
+++ TrackingProtection.txt	2019-04-23 16:46:27.641439201 -0700
@@ -1,4 +1,4 @@
-nb_trackers = 2061
+nb_trackers = 2060
 103092804.com
 1q2w3.website
 1rx.io
@@ -1443,7 +1443,6 @@
 payhit.com
 paymentsmb.com
 paypopup.com
-pcache.alexa.com
 pch.com
 peer39.com
 peer39.net
@@ -2061,7 +2060,7 @@
 zymerget.faith
 zypmedia.com
 
-nb_first_parties = 1122
+nb_first_parties = 1118
 24smi.net: 24smi.net
 2leep.com: 2leep.com
 33across.com: 33across.com
@@ -2225,7 +2224,6 @@
 aivalabs.com: aivalabs.com
 akamai.com: imiclk.com
 albacross.com: albacross.com
-alexa.com: pcache.alexa.com
 allstarmediagroup.com: allstarmediagroup.com
 aloodo.com: aloodo.com
 amadesa.com: amadesa.com
@@ -2372,7 +2370,6 @@
 clixtell.com: clixtell.com
 clovenetwork.com: clovenetwork.com
 clustrmaps.com: clustrmaps.com
-cnet.com: 2mdn.net
 cnzz.com: cnzz.com
 cognitivematch.com: cmads.com.tw,cmadsasia.com,cmadseu.com,cmmeglobal.com,cognitivematch.com
 coinhive.com: ad-miner.com,authedmine.com,bmst.pw,cnhv.co,coin-hive.com,coinhive.com,wsservices.org
@@ -2723,7 +2720,6 @@
 mediaforge.com: mediaforge.com
 mediaimpact.sk: azetklik.sk,rsz.sk
 medialets.com: medialets.com
-mediamarkt.at: google-analytics.com
 mediamath.com: adroitinteractive.com,designbloxlive.com,mathtag.com,mediamath.com
 mediametrie-estat.com: estat.com,mediametrie-estat.com
 mediaocean.com: adbuyer.com,mediaocean.com
@@ -2946,7 +2942,6 @@
 sap.com: seewhy.com
 sapient.com: bridgetrack.com,sapient.com
 sas.com: aimatch.com,sas.com
-saturn.at: google-analytics.com
 scandinavianadnetworks.com: scandinavianadnetworks.com
 scribol.com: scribol.com
 searchforce.com: searchforce.com,searchforce.net
```

I tested saturn.at and mediamarkt.at in Beta, both in Desktop and in responsive iPhone5 mode:

- homepage
- product accordion
- product details page
- zooming into a product photo
- adding a product to the shopping cart
- viewing the shopping cart

Google Analytics was not blocked by Shields, even without our explicit whitelist, but Google Tag Manager was.

I tested `cnet.com` both in Desktop and in responsive iPhone5 mode:

- homepage
- opening an article
- viewing a video

Everything was successful except the videos on mobile. The reason is not `2mdn.net` though, it's because of `pubads.g.doubleclick.net`.

### CODE REVIEW

File: data/disconnect.json [hunk 1]
```diff
@@ -10747,35 +10755,7 @@
...
-            "google-analytics.com"
-          ]
-        }
-      },
-      {
-        "Mediamarkt": {
-          "https://www.mediamarkt.at/": [
-            "google-analytics.com"
-          ]
-        }
```
Comment (SergeyZhukovsky) [inline data/disconnect.json]: the two above failed on Android, I'm not aware did we have problems on desktop or not, we didn't on desktop iirc. The tests should be done on Android to verify that correctly. I don't know any ways how that can be done except put it to production and try or use a rooted phone and replace the file.

Comment (fmarier) [reply data/disconnect.json]: I tested the mobile versions of the site (using the devtools), which is probably as much as we can do before going to production as you say.

I don't have a rooted phone to test this with. Maybe we should push to prod and then quickly test using a real Android phone?

Comment (SergeyZhukovsky) [reply data/disconnect.json]: Sounds good and revert quickly back if we see problems on them.

**Comment by fmarier** [comment]
I tested this on Android and of the 3 sites that were previously broken, only https://saturn.at is still a problem. I have filed https://github.com/brave/brave-browser/issues/4248 for it.

**Pull request merged**